src/mem/cache/tags/sector_tags.cc

```diff
@@ -360,7 +358,7 @@ SectorTags::regenerateBlkAddr(const CacheBlk* blk) const
     const SectorSubBlk* blk_cast = static_cast<const SectorSubBlk*>(blk);
     const SectorBlk* sec_blk = blk_cast->getSectorBlock();
     const Addr sec_addr =
-        indexingPolicy->regenerateAddr(blk->getTag(), sec_blk);
+        indexingPolicy->regenerateAddr({blk->getTag(), false}, sec_blk);
```

Comment:
This also looks like a bug. Why is it not using blk->isSecure()?

Reply:
Done